packages/js/src/TelnyxRTC.ts

```diff
@@ -1,3 +1,47 @@
 import TelnyxRTCClient from './Modules/Verto';
+import { ITelnyxRTCOptions } from './Modules/Verto/util/interfaces';
 
-export default class TelnyxRTC extends TelnyxRTCClient {}
+export default class TelnyxRTC extends TelnyxRTCClient {
+  /**
+   * Creates a new `TelnyxRTC` instance with the provided options.
+   *
+   * @param options An object with options.
+   * @param options.login_token The JSON Web Token (JWT) to authenticate with your SIP Connection. This is the recommended authentication strategy. [See how to create one](https://developers.telnyx.com/docs/v2/webrtc/quickstart).
```

Comment:
Nice, maybe verify and mention that `login_token` will take precedence over `login`/`password`?